frame/example-offchain-worker/src/lib.rs

```diff
@@ -269,19 +303,19 @@ impl<T: Trait> Module<T> {
 		// local keystore with expected `KEY_TYPE`.
 		let results = signer.send_signed_transaction(
 			|_account| {
-				let mut call_data = REPORT_METRICS_SELECTOR.to_vec();
-				call_data.extend_from_slice(&[
-					1,
-					2,
-					3,
-					4,
-				]);
+
+				let mut test_call_param = CallData::new();
+				test_call_param.push_arg(&REPORT_METRICS_SELECTOR);
```

Comment:
That does not seem correct, it is going to encode the vector of the selector, that’s not the same as copying it at the start of the buffer.